internal/namespaces/instance/v1/command_server_create.go

```diff
@@ -381,7 +400,11 @@ func buildVolumeTemplate(api *instance.API, zone scw.Zone, orgID, flagV string)
 		return buildVolumeTemplateFromUUID(api, zone, parts[0])
 	}
 
-	return nil, fmt.Errorf("Invalid volume format '%s'.", flagV) // TODO: improve msg [APIGW-1371]
+	return nil, &core.CliError{
+		Err:     fmt.Errorf("Invalid volume format '%s'.", flagV),
+		Details: "",
+		Hint:    `You must provide either a UUID ("7a892c1a-bbdc-491f-9974-4008e3708664"), a local volume size ("local:100G" or "l:100G") or a block volume size ("block:100G" or "b:100G").`,
```

Comment:
we should make it explicit this is an example UUID

use `xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx`